### PR TITLE
Add POC for system diagnostics

### PR DESCRIPTION
**Pull request by mathew-fleisch**

### CODE REVIEW

File: commands/reports.py [hunk 1]
```diff
@@ -102,11 +108,48 @@ def generate_gainers_report_card():
...
+  logger.info("generate_diagnostic_card()")
+  arows = []
+  brows = []
+  crows = []
+  rows = []
+  with os.popen("hostname") as line:
+    arows = line.readlines()
+  with os.popen("make --no-print-directory version") as line:
+    brows = line.readlines()
+  with os.popen("df -h") as line:
```
Comment (zmattingly) [inline commands/reports.py:137]: Looks like you can use `shutil` for this: https://docs.python.org/3/library/shutil.html#shutil.disk_usage

File: commands/reports.py [hunk 2]
```diff
@@ -102,11 +108,48 @@ def generate_gainers_report_card():
...
+  db.close()
+  return results
+
+def generate_diagnostic_card():
+  logger.info("generate_diagnostic_card()")
+  arows = []
+  brows = []
+  crows = []
+  rows = []
+  with os.popen("hostname") as line:
```
Comment (zmattingly) [inline commands/reports.py:133]: I think you can `import platform` and use 

```
platform.node()
```

for this

File: commands/reports.py [hunk 3]
```diff
@@ -102,11 +108,48 @@ def generate_gainers_report_card():
...
+    arows = line.readlines()
+  with os.popen("make --no-print-directory version") as line:
```
Comment (zmattingly) [inline commands/reports.py:135]: If we're just loading the yaml file as text I think you could use the regular `open` from python and iterate over the lines using a regex to parse out what you want.

**Review by zmattingly — COMMENTED** [review]
Couple suggestions if you want to avoid shelling out. I think what you're doing is fine though tbh.

**Review by jp00p — APPROVED** [review]

**Pull request merged**